### ISSUE TITLE
pythondistdeps attempts to do invalid rich Provides (rpm 4.15.1 with backported fixes from master)

### ISSUE THREAD

**Issue by Conan-Kudo** [rpm-software-management/rpm#981]
When bootstrapping RPM 4.15.1 with pythondistdeps fixes backported in OpenMandriva Cooker, I've encountered the following bug:

```
BUILDSTDERR: error: No rich dependencies allowed for this type: (python3.8dist(createrepo-c) == 0.15.4)
BUILDSTDERR: error: No rich dependencies allowed for this type: (python3dist(createrepo-c) == 0.15.4)
```

I'm pretty certain the issue comes from 403f75cdbb94ced52f169684c12ee5110e468dfa. It should be easily fixable...

**Comment by Conan-Kudo** [comment]
cc: @gordonmessmer, @hroncok

**Comment by hroncok** [comment]
When the list contains just one item, the parenthesis should not be there, correct?

**Comment by Conan-Kudo** [comment]
Yes, but also you're not allowed to do rich Provides at all

**Comment by ignatenkobrain** [comment]
> but also you're not allowed to do rich Provides at all

… which would be so amazing, wouldn't it? :)

**Comment by hroncok** [comment]
oh, this is for provides?

**Comment by Conan-Kudo** [comment]
@hroncok Yep. But I'm pretty sure you can't do this exact thing on Requires either...

**Comment by Conan-Kudo** [comment]
This is fixed with the merge of #982.
